docs/tests/test-flags.md

```diff
@@ -50,7 +51,13 @@ There are two ways to do this:
   end
 ```
 
-**NOTE** Any specs that use the `js: true` flag might be hitting Elasticsearch.
-You may want to consider clearing out Elasticsearch data for those specs even if
-you don't intend on working with Elasticsearch directly to ensure that you have
-a clean page load with no unexpected data.
+### `js: true` Flag
+
+`js: true` indicates to our specs that we want the javascript on the page to be
+execute when the page is rendered. One side effect of running our javascript is
+that a lot of pages will hit Elasticsearch. Since we don't clean out
+Elasticsearch between every single spec(because it is very costly) this can lead
+to unexpected data being loaded for a spec. To prevent this use the
+`:stub_elasticsearch` flag. The `:stub_elasticsearch` flag will stub all index
+and search requests made to Elasticsearch and return an empty response. This
+will ensure that no unwanted data shows up on your page.
```

Comment:
```suggestion
will ensure that no unwanted data shows up on your spec's page.
```

i think it's worth clarifying this because this is referencing the capybara `page` object when running `js: true` tests, right?

Reply:
yep!